src/rx_signal_processing.py

```diff
@@ -277,12 +270,17 @@ def correlations_from_samples(beamformed_samples_1, beamformed_samples_2, output
             row = samples_for_all_range_lags[..., 1].astype(np.int32)
 
             # [num_range_gates, num_lags, 2]
-            column = samples_for_all_range_lags[..., 0].astype(np.int32)
+            col = samples_for_all_range_lags[..., 0].astype(np.int32)
+
+            values_for_slice = np.empty((beamformed_samples_1.shape[1], row.shape[0], row.shape[1]), dtype=np.complex64)
 
-            values_for_slice = correlated[s['slice_num'], :, row, column]
+            for lag in range(row.shape[1]):
+                values_for_slice[:, :, lag] = np.einsum('ij,ij->ji',
+                                                        beamformed_samples_1[s, :, row[:, lag]],
+                                                        beamformed_samples_2[s, :, col[:, lag]].conj())
 
-            # [num_range_gates, num_lags, num_beams]
-            values_for_slice = np.einsum('ijk,j->kij', values_for_slice, s['lag_phase_offsets'])
+            # [num_beams, num_range_gates, num_lags]
```

Comment:
Won't changing this data order have major ramifications in down stream processing?

Reply:
The data order of `values_for_slice` was also changed before this line, so the end result is the same data order.